third_party/terraform/resources/resource_google_project_services.go

```diff
@@ -148,19 +149,21 @@ func reconcileServices(cfgServices, apiServices []string, config *Config, pid st
 		return sm
 	}
 
+	sort.Strings(cfgServices)
 	cfgMap := m(cfgServices)
+	log.Printf("[DEBUG]: Saw the following services in config: %v", cfgServices)
 	apiMap := m(apiServices)
+	log.Printf("[DEBUG]: Saw the following services enabled: %v", apiServices)
 
 	for k := range apiMap {
 		if _, ok := cfgMap[k]; !ok {
-			// The service in the API is not in the config; disable it.
-			err := disableService(k, pid, config)
+			log.Printf("[DEBUG]: Disabling %s as it's enabled upstream but not in config", k)
```

Comment:
I think this log is more helpful if embedded inside `disableService`

Reply:
`disableService` already logs the request itself, and it's used in other places where this message text isn't relevant; this message attaches additional context only in this block.